AcademyChallenge/ViewModel/AvatarListViewModel.swift

```diff
@@ -17,7 +17,16 @@ class AvatarListViewModel {
         }
         return avatarService.fetchAvatar()
     }
-    //    func deleteAV(avatar: Avatar) {
-    //        self.avatarService?.deleteAvatar(avatarToDelete: avatar)
-    //    }
+        func deleteAV(avatar: Avatar) -> Completable {
```

Comment:
what is AV? `delete(avatar: Avatar)` looks more legible in this case

Reply:
solved